### ISSUE TITLE
Group name in combined display

### ISSUE THREAD

**Issue by cometkim** [evanwashere/mitata#38]
First, thanks for the response #31 

I tried using it to reproduce the output of version 0.x

```js
group('group name', () => {
  summary(() => {
    ...
  })
})
// group name is not displayed

summary(() => {
  group('group name', () => {
  })
})
// group name is displayed well in the header, but not summary like in v0.x
```

**Comment by evanwashere** [comment]
nested nodes not inheriting group name from root node is fixed in ^1.0.15
as for showing group names in the summary itself, they were removed in v1 for brevity and there are no plans to bring it back.

**Comment by cometkim** [comment]
That was quick! thanks :)
